eve/main.yml

```
@@ -285,6 +286,144 @@ stages:
           doStepIf: false
           alwaysRun: true
 
+  single-node-upgrade-test:
+    worker:
+      type: openstack
+      image: CentOS-7-x86_64-GenericCloud-1809.qcow2
+      flavor: m1.large
+      path: eve/workers/openstack-single-node
+    steps:
+      - Git: *git_pull
+      - ShellCommand:
```

Replace this block with 
```suggestion
      - ShellCommand: *ssh_ip_setup
```